### PR TITLE
JUCX: fix rpmbuild errors.

### PR DESCRIPTION
**Pull request by petro-rudenko**
To fix issues in https://github.com/openucx/ucx/pull/4362

### CODE REVIEW

File: ucx.spec.in [hunk 1]
```diff
@@ -291,6 +292,13 @@ process to map the memory of another process into its virtual address space.
 %{_libdir}/ucx/libuct_xpmem.so.*
 %endif
 
+%description jucx
```
Comment (amaslenn) [inline ucx.spec.in]: Don't you need to add something like this:
```
%if %{with jucx}
%package jucx
Requires: %{name}%{?_isa} = %{version}-%{release}
Summary: ...
Group: System Environment/Libraries

%description
...
```
?

File: ucx.spec.in [hunk 2]
```diff
@@ -115,6 +115,7 @@ rm -f %{buildroot}%{_libdir}/ucx/lib*.so
 rm -f %{buildroot}%{_libdir}/ucx/lib*.a
 
 %files
+%{_libdir}/jucx-*.jar
```
Comment (amaslenn) [inline ucx.spec.in]: What if UCX was built without Java support?

File: ucx.spec.in (hunk 1 above)
Comment (petro-rudenko) [reply ucx.spec.in]: Removed this for now.

File: ucx.spec.in [hunk 3]
```diff
@@ -13,6 +13,7 @@
 %bcond_with    rocm
 %bcond_with    ugni
 %bcond_with    xpmem
+%bcond_with    java
```
Comment (yosefe) [inline ucx.spec.in:16]: this means the default would be to build without JAVA
it should be `%bcond_without` instead - pls fix in UCX master

**Pull request merged**